### ISSUE TITLE
Refactor and merge the two "send_image" methods

### ISSUE THREAD

**Issue by aitorres** [aitorres/dogpicsbot#7]
The bot's two methods that take care of sending pictures ([send_dog_picture](https://github.com/aitorres/dogpicsbot/blob/b33b12c1960294160c1c7b5418c0112ce22c06c0/bot.py#L224) and [send_fox_picture](https://github.com/aitorres/dogpicsbot/blob/b33b12c1960294160c1c7b5418c0112ce22c06c0/bot.py#L256)) are very similar. The duplicate logic should be moved to a separate function called in both places, or otherwise refactored so that we don't duplicate code.

Besides this change, it would be a bonus if all the calls to `context.bot.send_photo` would reply to the received `message_id` :-)

**Comment by kana800** [comment]
hey, I am beginner looking to contribute and I can work on this if its up for grab
Do you want to merge `send_dog_picture` and `send_fox_picture`

**Comment by seleregb** [comment]
Hey @aitorres I can take this once you're good with my PR for the PEP8 style changes

**Comment by aitorres** [comment]
> Hey @aitorres I can take this once you're good with my PR for the PEP8 style changes

Sure thing @seleregb!

**Comment by seleregb** [comment]
@aitorres  I'm getting this error when setting up the environment. Are you okay with me changing the library versions?

```
ERROR: Cannot install -r requirements.txt (line 23) and urllib3==1.26.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested urllib3==1.26.5
    requests 2.24.0 depends on urllib3!=1.25.0, !=1.25.1, <1.26 and >=1.21.1
```

**Comment by aitorres** [comment]
> @aitorres I'm getting this error when setting up the environment. Are you okay with me changing the library versions?
> 
> ```
> ERROR: Cannot install -r requirements.txt (line 23) and urllib3==1.26.5 because these package versions have conflicting dependencies.
> 
> The conflict is caused by:
>     The user requested urllib3==1.26.5
>     requests 2.24.0 depends on urllib3!=1.25.0, !=1.25.1, <1.26 and >=1.21.1
> ```

Yes, go ahead and thanks for letting me know! I think if you chage `requests` to `2.26.0` the problem will be fixed.

**Comment by aitorres** [comment]
Closed by #14
